src/common/mime.cc

```diff
@@ -239,9 +321,14 @@ namespace Pistache::Http::Mime
 
                 if (match_literal('=', cursor))
                 {
-                    double val;
-                    if (!match_double(&val, cursor))
```

Comment:
The `match_double` function is now unused, so it should probably go, as  I don't see the use for that locale-dependent implementation in the future.

Reply:
You're right, but `match_double` is also part of the public API, and it may be used by someone already

Reply:
In which case, it's worth mentionning in the doc that it's locale dependent :)